src/routes/analysis.py

```
@@ -3,273 +3,390 @@
-                'difficulty_order': sorted(analysis_result['concepts'], key=lambda x: x['importance'], reverse=True),
-                'estimated_completion_weeks': max(1, total_study_time // (7 * 60))  # Basé sur 1h/jour
+                'spaced_repetition': spaced_repetition,
             },
-            'timestamp': datetime.now().isoformat()
-        }
-        
-        return jsonify(response)
-        
+            'timestamp': datetime.now().isoformat(),
+        })
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)